### PR TITLE
Finished read methods in ControlNetVersioner

### PR DESCRIPTION
**Pull request by jessemapel**

### CODE REVIEW

File: isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp [hunk 1]
```diff
@@ -792,6 +788,104 @@ namespace Isis {
...
+    catch (IException &e) {
+      QString msg = "Failed to parse the header from the protobuf control network file.";
+      throw IException(e, IException::User, msg, _FILEINFO_);
+    }
+
+    // Create the control points
+    for (int i = 0; i < protoNet.points_size(); i++) {
+      try {
+        QSharedPointer<ControlNetFileProtoV0001_PBControlPoint>
+              protoPoint(mutable_points.points(i));
```
Comment (kberryUSGS) [inline isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Should this be something like: 
`protoPoint(protoNet.mutable_points(i))`  or `protoPoint(protoNet.mutable_points().points(i)`

I couldn't find where `mutable_points` was defined to be usable this way in `ControlNetFileV0001.pb.h` but I may just be confused searching through the generated code.

Comment (jessemapel) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: It should be protoNet.mutable_points

Comment (kberryUSGS) [reply isis/src/control/objs/ControlNetVersioner/ControlNetVersioner.cpp]: Okay, :+1:

**Comment by kberryUSGS** [comment]
Looks good!

**Pull request merged**